third-party/llvm/Makefile

```diff
@@ -36,6 +36,7 @@ $(LLVM_FILE):
 	cd $(LLVM_BUILD_DIR) && $(MAKE)
 	cd $(LLVM_BUILD_DIR) && $(MAKE) install
 	cd $(LLVM_BUILD_DIR) && cd test && $(MAKE) lit.site.cfg
+	../../util/config/gather-clang-sysroot-arguments > configured-clang-sysroot-arguments
```

Comment:
I think I will look at putting it in the third-party/llvm/install dir, if that's OK with you.